muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/user_credentials.ts

```diff
@@ -0,0 +1,11 @@
+import notifications from './base/notifications';
+import locales from './base/locales';
+import i18n from './base/i18n';
+// import forgotPassword from './base/forgot-password';
+import {combineReducers} from 'redux';
+
```

Comment:
can we change the name to user-credentials.ts?

If we have been using that naming convention.